.github/workflows/ci.yaml

```diff
@@ -131,3 +132,71 @@ jobs:
         with:
           name: kind-logs-${{ matrix.ip-family }}-${{ matrix.bgp-type }}-${{ matrix.deployment}}
           path: /tmp/kind_logs/
+
+  e2e-use-operator:
+    runs-on: ubuntu-20.04
+    defaults:
+      run:
+        shell: bash
+        working-directory: metallb
+    strategy:
+      fail-fast: false
+    steps:
+      - name: Checkout Metal LB Operator
+        uses: actions/checkout@v2
+        with:
+          repository: metallb/metallb-operator
+          path: metallboperator
+          ref: d5836e27522a54c3e022fde53c1f52a1446059c4
+
+      - uses: actions/setup-go@v2
+        with:
+          go-version: '1.16.3'
+
+      - name: Checkout MetalLB
```

Comment:
Do we need to do this? I'd expect MetalLB to be already available.

Reply:
This is the first time metallb being checked out. do you want to do something like  
[    steps:
      - name: Checkout
        uses: actions/checkout@v2
        with:
          fetch-depth: 0](https://github.com/metallb/metallb/blob/31080d6f4e06f7d579127034b8ecc28e5ca702d4/.github/workflows/ci.yaml#L85-L89)
at the start of the job ?

Reply:
Ah ignore me, I just noticed the other job does that as well